hub/policy/src/integration-test/java/uk/gov/ida/integrationtest/hub/policy/apprule/MatchingServiceResourcesIntegrationTest.java

```diff
@@ -101,6 +101,7 @@
     private SamlResponseWithAuthnRequestInformationDto translatedAuthnRequest;
     private SamlAuthnRequestContainerDto rpSamlRequest;
     private boolean registering = true;
+    private LevelOfAssurance requestedLoa = LevelOfAssurance.LEVEL_2;
```

Comment:
this (and the line above) could be `static final`, but as above, perhaps it doesn't need to exist at all?